src/fluent-plugin-mdsd/gemspec-template

```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+    s.name = "fluent-plugin-mdsd"
+    s.version = "GEMVERSION"
+    s.date = "2017-01-05"
```

Comment:
>01-05 [](start = 19, length = 5)

Should this date be updated to be today (2017-05-01)? Or is this supposed to be unchanged since it was first created back in January?